client/src/actions/team.js

```diff
@@ -224,14 +272,15 @@ function teamDataAllocator(teamData) {
     orgName: teamData.orgName,
     twitterHandle: teamData.twitterHandle,
     repoCreated: teamData.repoCreated,
+    error: null,
```

Comment:
Sorry I might have missed it while reviewing, but what is this error attribute used for?

Reply:
I think this is part of the discussion with Rio about whether we should handle the errors globally or within the related states, so error is actually in the initial team state. But I noticed that this teamDataAllocator function doesn't return error, so the state ends up not having the error field since the reducer just returns action.payload (which is what is returned by this function)